model/facade/action/GetHolidayHoursBaseAction.php

```diff
@@ -124,8 +124,11 @@ protected function getHoursSummary(DateTime $referenceDate = NULL): array
 
                 // We strictly get the holidays spent between the dates specified in
                 // the report, not until the end of the year (see #352).
-                $vacations = $taskDao->getVacations($userVO, $reportInit, $reportEnd);
-                $vacations = $vacations["add_hours"] ?? 0;
+                $vacations_sum = $taskDao->getVacations($userVO, $reportInit, $reportEnd);
+                $vacations = $vacations_sum["add_hours"] ?? 0;
+                $updated_at = "";
+                if ($vacations_sum && $vacations_sum["updated_at"])
+                    $updated_at = str_split($vacations_sum["updated_at"], 10)[0];
```

Comment:
It's probably easier/more readable to use `substr($vacations_sum["updated_at"], 0, 10)`.

Reply:
Much better, thanks!